AdNetworkSupport/HyprMX/HyprMXRewardedVideoCustomEvent.m

```diff
@@ -45,45 +57,62 @@ - (void)requestRewardedVideoWithCustomEventInfo:(NSDictionary *)info {
     
     [HyprMXController initializeSDKWithDistributorId:distributorID userID:userID];
     
-    [HyprMXController canShowAd:^(BOOL isOfferReady) {
-        if (isOfferReady) {
-            [self.delegate rewardedVideoDidLoadAdForCustomEvent:self];
-
-        } else {
-            [self.delegate rewardedVideoDidFailToLoadAdForCustomEvent:self error:nil];
-        }
-    }];
-}
-
-- (BOOL)hasAdAvailable {
-    return [HyprMXController hasAdAvailable];
+    [self.rewardedPlacement loadAd];
 }
```

Comment:
You lost a method `- (BOOL)hasAdAvailable` .
That is a mopub required method you need to implement, which just needs to return the result of `[self.rewardedPlacement isAdAvailable]`